scripts/belyi/find_friends.py

```diff
@@ -0,0 +1,104 @@
+from lmfdb import db
+from lmfdb.ecnf.WebEllipticCurve import parse_ainvs
+from lmfdb.belyi.main import hyperelliptic_polys_to_ainvs, curve_string_parser
+from scripts.ecnf.import_utils import NFelt
+from sage.all import EllipticCurve
+import re
+from lmfdb.genus2_curves.main import genus2_lookup_equation as genus2_lookup_equation_polys
+
+def genus2_lookup_equation(rec):
+    f,h = curve_string_parser(rec)
+    return genus2_lookup_equation_polys(str([f,h]))
+
+def genus1_lookup_equation_QQ(rec):
+    assert rec['g'] == 1
+    f,h = curve_string_parser(rec)
+    ainvs = hyperelliptic_polys_to_ainvs(f,h)
+    E = EllipticCurve(ainvs)
+    j = E.j_invariant()
+    for r in db.ec_curves.search({"jinv":str(j)}):
+        ainvs2 = r['ainvs']
+        E2 = EllipticCurve(ainvs2)
+        if E.is_isomorphic(E2):
+            return r['lmfdb_label']
+    #print("Curve not found in database")
+    return None
+
+def genus1_lookup_equation_nf(rec):
+    assert rec['g'] == 1
+    # make base field
+    nf_matches = list(db.nf_fields.search({'coeffs':rec['base_field']}))
```

Comment:
Is `coeffs` always in polredabs form?

Reply:
Yes, we always store the polredabs coefficients.